monitoring/onpremise/fluent-bit/main.tf

```diff
@@ -149,3 +149,133 @@ resource "kubernetes_daemonset" "fluent_bit" {
     }
   }
 }
+
+resource "kubernetes_daemonset" "fluent_bit_windows" {
+  count = (local.fluent_bit_windows_is_daemonset && length(var.node_selector_windows) > 0) ? 1 : 0
```

Comment:
Maybe you should add a condition on whether a windows node is needed

Reply:
That is already the case with the count